README.md

```diff
@@ -56,6 +56,27 @@ go install github.com/hyperledger/firefly-cli/ff@latest
 $ ff init <stack_name>
 ```
 
+### Stack Initialization Options
+
+When initializing a new stack, you can configure various options including:
+
+```
+--from-block      For multiparty networks only: specify the starting block for event processing. 
```

Comment:
Should only work with `remote-node-url` as with the local chain that the CLI creates we want to start from 0

Reply:
as it should point to when the contract was deployed - I'm more tempted for this to be `--contract-block` as you pass in the MultiParty Contract address through `--contract`